frontend/app/src/pages/room/index.tsx

```diff
@@ -0,0 +1,24 @@
+import { Page } from "@/components"
+import { ROUTE } from "@/constants/route";
+import { Collaborator, InfoBar } from "@/features/room"
+import { VStack } from "@chakra-ui/react"
+import { useParams, Navigate } from "react-router-dom";
+
+function RoomPage() {
+
+    const { roomId } = useParams();
+
+    console.log(roomId)
```

Comment:
remove unused console log

Reply:
Fixed in [cc32e0c](https://github.com/CS3219-AY2324S1/ay2324s1-course-assessment-g03/pull/18/commits/cc32e0c13b9ae764e5b33d9fe0384144f797a307)